solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,76 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# Make 'run-all' to test the other tests in these files.
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+TEST = pandas/tests/arrays/string_/test_string.py
+PACKAGE_PATH = myst/bin/python3
+PYTEST_INI_FILE = pytest.ini
+
+OPTS = --nobrk --max-affinity-cpus=4
```

Comment:
"NoBRK" is supported in config.json now. Move it to config.json, as there's ongoing effort to enforce --nobrk according to config.json, even if the the entry is not present in config.json (treated as "NoBRK"=false).